### PR TITLE
Vp 168 uncaught exceptions when running tests on windows

### PR DESCRIPTION
**Pull request by avowkind**
Cause:  mongoose disconnect. 

we are running tests in parallel.  Although we now use mongo-memory-server to give us multiple parallel databases mongoose is still a single thing.  So mongoose connect connects to different ports but when we call mongoose.disconnect it drops everything.   
Commenting out and the tests run ok.  there may be dangling connections but the tests don't care. 

Tested on Windows.

### CODE REVIEW

File: server/api/person/__tests__/person.spec.js [hunk 1]
```diff
@@ -4,20 +4,17 @@ import { server, appReady } from '../../../server'
...
-  await memMongo.start()
+  await t.context.memMongo.start()
 })
 
-test.after.always(async () => {
-  await memMongo.stop()
-  // console.log('stopped')
+test.after.always(t => {
+  t.context.memMongo.stop()
+  console.log('stopped')
```
Comment (michael-j-chinn) [inline server/api/person/__tests__/person.spec.js]: Should we comment out or remove this log?

**Comment by michael-j-chinn** [comment]
What are the ramifications of those hanging connections? How long for them to get cleaned up? Is there any concern of resource exhaustion during heavy repeated test running?

File: server/api/person/__tests__/person.spec.js (hunk 1 above)
Comment (avowkind) [reply server/api/person/__tests__/person.spec.js]: should remove eventually.

**Comment by avowkind** [comment]
I think the connections will close fairly soon anyway after the test goes out of scope and the GC runs. As we don't have this problem on the mac I suspect it is just a timing issue.

**Pull request merged**